stylus-sdk/src/abi/export.rs

```diff
@@ -40,6 +40,8 @@ pub fn print_abi<T: GenerateAbi>() {
     println!(" * For more information, please see [The Stylus SDK](https://github.com/OffchainLabs/stylus-sdk-rs).");
     println!(" */");
     println!();
+    println!("// SPDX-License-Identifier: TODO");
```

Comment:
We should either parse the toml for this or drop the line imo. Otherwise you're letting the consumer of the crate decide the license of the abi in a way.

Reply:
Makes sense, will drop this line

Reply:
Will do some investigation into parsing the toml to get rid of the pesky warning and use the right license here